tmail-backend/tmail-third-party/openpaas/src/main/java/com/linagora/tmail/mailet/CalDavCollect.java

```diff
@@ -146,7 +154,7 @@ private Mono<Void> updateDavCalendar(DavCalendarObject davCalendarObject, Calend
                 davCalendarObject.eTag());
             return davClient.doUpdateCalendarObject(davUser.username(), newDavCalendarObject);
         } else {
-            return Mono.empty();
+            return Mono.error(new CalDavWarningException("Data in the ics file is older than the current one in dav server"));
```

Comment:
Mono.error(new OutdatedUpdateException("Data in the ics file is older than the current one in dav server"));